client/src/connection/session.ts

```diff
@@ -30,7 +30,7 @@ export abstract class Session {
   }
 
   protected abstract establishConnection(): Promise<void>;
-  abstract run(code: string): Promise<RunResult>;
+  abstract run(code: string, skipPageHeaders?: boolean): Promise<RunResult>;
```

Comment:
I'm not sure do we really want to change the base signature? `skipPageHeaders` isn't implemented in other connection methods.
how about just `await (session as ITCSession).run(code, true);` in CodeRunner?

Reply:
Done in https://github.com/sassoftware/vscode-sas-extension/pull/880/commits/c9ca53cf966b6544f4769897964db5d4bf12fe76